Firebase/Core/Public/FirebaseCore.h

```diff
@@ -18,4 +18,8 @@
 #import "FIRApp.h"
 #import "FIRConfiguration.h"
 #import "FIRLoggerLevel.h"
+#import "FIRObjectSwizzler.h"
 #import "FIROptions.h"
+#import "FIRSwizzledObject.h"
+#import "FIRSwizzler.h"
```

Comment:
Are all three headers supposed to be public outside of Firebase?

Note how the Core subspec sets public_header_files and private_header_files. The private_header_files are still available to pod consumers, but only by being explicitly included.

Reply:
I think the headers should be private in FIRCore--though it won't matter if the swizzler is never merged to master before being factored into its own pod.

Reply:
Done.